src/providers/bedrock/index.ts

```diff
@@ -56,108 +40,89 @@ import {
 const BedrockConfig: ProviderConfigs = {
   api: BedrockAPIConfig,
   getConfig: (params: Params) => {
-    if (!params.model) {
-      throw new GatewayError('Bedrock model not found');
-    }
-
-    // To remove the region in case its a cross-region inference profile ID
-    // https://docs.aws.amazon.com/bedrock/latest/userguide/cross-region-inference-support.html
-    const providerModel = params.model.replace(/^(us\.|eu\.)/, '');
```

Comment:
This is important to support cross region model IDs. Please add the whole code block back.